pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -586,6 +632,11 @@ func (r *reconciler) update(lastRealized *lastRealized, newRule *CompletedRule,
 			lastRealized.podOFPorts[svcKey] = newOFPorts
 		}
 	} else {
+		if r.fqdnController != nil && len(newRule.To.FQDNs) > 0 {
+			if err := r.fqdnController.addFQDNRule(newRule.ID, newRule.To.FQDNs, r.getOFPorts(newRule.TargetMembers)); err != nil {
+				klog.ErrorS(err, "Error when adding FQDN rule", "ruleID", newRule.ID)
```

Comment:
this should probably return the error instead of logging it since `update` can return an error